src/main/java/com/cleanroommc/modularui/test/TestTile.java

```diff
@@ -270,7 +276,27 @@ public ModularPanel createPanel(GuiContext context) {
                                                         .texture(GuiTextures.CHECK_BOX)
                                                         .size(14, 14))
                                                 .child(IKey.lang("bogosort.gui.enabled").asWidget()
-                                                        .height(14))))));
+                                                        .height(14)))))
+                        .addPage(
+                            new ParentWidget<>()
+                                .size(1f, 1f)
+                                .padding(7)
+                                .child(SlotGroupWidget.playerInventory())
+                                .child(
+                                    dropDown
+                                        .addChoice(i -> {
+                                            currentDropdownIndex = dropDown.getSelectedIndex();
+                                            markDirty();
```

Comment:
`markDirty` here doesn't do anything as widgets are client-only. We should port https://github.com/GTNewHorizons/ModularUI/commit/1e9d12a6a017ed3e33067ef6ac0adb3064fbf818 but it's out of the scope of this PR.

Reply:
ah. I see. Should stop working on code when I am sleepy lmao.